### PR TITLE
New Adapter: AndBeyondMedia

### PR DESCRIPTION
**Pull request by AndBeyondMediaHB**
doc - https://github.com/prebid/prebid.github.io/pull/3899

### CODE REVIEW

File: static/bidder-params/andbeyondmedia.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Compass Adapter Params",
```
Comment (SyntaxNode) [inline static/bidder-params/andbeyondmedia.json:3]: Is this an alias of the Compass adapter? If so, we can advise on better ways to create the alias.

Comment (AndBeyondMediaHB) [reply static/bidder-params/andbeyondmedia.json]: @SyntaxNode hello, this is not an alias, this is a separate adapter

**Comment by bsardo** [comment]
Hi @AndBeyondMediaHB. Thanks for your contribution. We're curious why you've copied the entire Compass adapter including config files and tests. We strongly prefer avoiding duplicate adapter code.

**Review by bsardo — COMMENTED** [review]
I see that you've copied over the `endpointId` parameter but it was omitted in the referenced docs PR. Can you please resolve this discrepancy?

**Comment by VeronikaSolovei9** [comment]
Local testing looks good.

File: static/bidder-params/andbeyondmedia.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "AndBeyond.Media Adapter Params",
+    "description": "A schema which validates params accepted by the Compass adapter",
```
[the remark this answers is not in the document]
Comment (AndBeyondMediaHB) [reply static/bidder-params/andbeyondmedia.json]: done

File: adapters/andbeyondmedia/params_test.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+var validParams = []string{
+	`{"placementId": "test"}`,
+	`{"placementId": "1"}`,
+	`{"endpointId": "test"}`,
+	`{"endpointId": "1"}`,
+}
+
+var invalidParams = []string{
+	`{"placementId": 42}`,
```
[the remark this answers is not in the document]
Comment (AndBeyondMediaHB) [reply adapters/andbeyondmedia/params_test.go:43]: done

File: static/bidder-info/andbeyondmedia.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+maintainer:
+  email: "sysengg@andbeyond.media"
+capabilities:
```
[the remark this answers is not in the document]
Comment (AndBeyondMediaHB) [reply static/bidder-info/andbeyondmedia.yaml:3]: thanks, we are not in the vendor list

**Review by VeronikaSolovei9 — APPROVED** [review]
LGTM

**Review by bsardo — APPROVED** [review]
LGTM

**Pull request merged**